### PR TITLE
Set free limit ratio as 1.0 at integration_node_test

### PR DESCRIPTION
**Pull request by ehgmsdk20**
Changes:
- Set env vars for testing
   * FREE_TX_POOL_SIZE_LIMIT_RATIO: 1.0
   * FREE_TX_POOL_SIZE_LIMIT_RATIO_PER_ACCOUNT: 1.0
- enable float env vars

Related issues:
#996

### CODE REVIEW

File: common/constants.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ function setNodeConfigs() {
     if (valFromEnvVar !== undefined) {
       if (CommonUtil.isBool(valFromNodeParams)) {
         NodeConfigs[param] = CommonUtil.convertEnvVarInputToBool(valFromEnvVar);
-      } else if (CommonUtil.isIntegerString(valFromEnvVar)) {
+      } else if (!isNaN(valFromEnvVar)) {
```
Comment (platfowner) [inline common/constants.js:106]: It looks like this change could be a little bit risky.
What about introducing a new util function like CommonUtil.isNumberString() that covers floating point numbers using regular expression?
regexp ref: https://github.com/ainblockchain/ain-blockchain/blob/master/db/state-util.js#L128

Comment (ehgmsdk20) [reply common/constants.js]: regex를 이용하여 isFloatString 함수 추가했습니다!

File: db/rule-util.js [hunk 2]
```diff
@@ -16,6 +16,10 @@ class RuleUtil {
     return this.isString(value) && /^-?\d+$/.test(value);
   }
 
+  isFloatString(value) {
```
Comment (platfowner) [inline db/rule-util.js:19]: Could you add test cases?
As this is a primitive function, it'd be safe with thorough test cases.
(please see other cases in rule-util.test.js)

Comment (ehgmsdk20) [reply db/rule-util.js:19]: Thanks for your comments! I added test code for isFloatString.

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Pull request merged**